Source/Initialization/PlasmaInjector.H

```diff
@@ -47,6 +48,7 @@ public:
 
     amrex::Real getCharge () {return charge;}
     amrex::Real getMass () {return mass;}
+    PhysicalSpecies getPhysicalSpecies() {return physical_species;}
```

Comment:
```suggestion
    PhysicalSpecies getPhysicalSpecies() const {return physical_species;}
```
I think that this member function could be declared as `const` . `getCharge` and `getMass` could be `const` as well